ui/window.go

```diff
@@ -159,7 +170,10 @@ func (win *Window) Loop(shutdown chan int) {
 			case wallet.BlockConfirmed:
 				win.updateSyncProgress(update.ConfirmedTxn)
 			}
-
+		case <-win.clipboardIsPasted:
+			win.window.ReadClipboard()
```

Comment:
When the clipboard channel receives the signal of the read type, the window.ReadClipBoard() method should be called. And when it receives a write type, the window.WriteClipBoard should be called with the text received on the channel.